src/main/resources/META-INF/jqassistant-rules/jsf/jakarta-jsf.xml

```diff
@@ -0,0 +1,20 @@
+<jqassistant-rules xmlns="http://schema.jqassistant.org/rule/v1.10"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://schema.jqassistant.org/rule/v1.10 http://schema.jqassistant.org/rule/jqassistant-rule-v1.10.xsd">
+
+    <concept id="jakarta-jsf:JSFManagedBeans">
+        <providesConcept refId="jsf:JSFManagedBeans"/>
+        <description>Labels all types annotated with @jakarta.faces.bean.ManagedBean with "Jsf" and "ManagedBean".</description>
+        <cypher><![CDATA[
+            MATCH
+              (t:Type)-[:ANNOTATED_BY]->()-[:OF_TYPE]-(a:Type)
+            WHERE
+              a.fqn="jakarta.faces.bean.ManagedBean"
+            SET
+              t:Jsf:ManagedBean
+            RETURN
+              t AS `JSF Managed Bean`
+    ]]></cypher>
+    </concept>
+    
+</jqassistant-rules>
```

Comment:
Gleiches Problem wie bei `@New`: In unserer Jakarta EE Version wird `@ManagedBean` nicht mehr unterstützt. Wollen wir die Regel trotzdem drin lassen? Dann können wir sie aktuell nicht testen.

Reply:
Sieht jemand eine Möglichkeit, das sinnvoll zu testen?